src/actions/createUser.js

```diff
@@ -0,0 +1,57 @@
+let fs = require("fs");
+let users = require("../../db/users.json")
+
+let createUser = (req, res) => {
+  let name = req.body;
+  if (dataValid(name) === 1) {
+    name.userId = idMax(users)+1;
+    users.push(name);
+    fs.writeFile("db/users.json", JSON.stringify(users), err => {
+      if (err) throw err(console.log("Nu s-a putut suprascrie"));
```

Comment:
check type of `err`. It most probably is an object, which means you can not use it as a function.